File: freecodecamp/2026-01-14.py
import re

def parse_link(markdown):
    # Extract link_text and link_url using regex
    match = re.match(r'\[([^\]]+)\]\(([^)]+)\)', markdown)
    
    if match:
        link_text = match.group(1)
        link_url = match.group(2)
        return f'<a href="{link_url}">{link_text}</a>'
    
    return markdown 

File: freecodecamp/2026-01-14.js
function parseLink(markdown) {
  const match = markdown.match(/\[([^\]]+)\]\(([^)]+)\)/);
  
  if (match) {
    const linkText = match[1];
    const linkUrl = match[2];
    return `<a href="${linkUrl}">${linkText}</a>`;
  }
  
  return markdown;
}
